### ISSUE TITLE
Use j/k to navigate select lists

### ISSUE THREAD

**Issue by muellerj** [piotrmurach/tty-prompt#22]
Using VIM-like keymaps for most other applications, it would be nice if the select offered a way to configure the up/down keys.

**Comment by piotrmurach** [comment]
As it turns out I cannot distinguish between for example `ctrl+j` and `enter` as both keypresses returns the same bytes sequence that converts to `\n` character. Similar story with `alt+j` which converts to similar byte sequence as many `alt` combinations. 

However, I found a way to allow you script your own behaviour with actual `j`, `k` keys like so:
```ruby
prompt = TTY::Prompt.new
prompt.on(:keypress) do |event|
  if event.key.name == 'j'
    prompt.publish(:keydown)
  end
  etc....
end
prompt.select( .... )
```
I don't want to add this behaviour as default because prompts such as `expand`, `enum_select` allow you to map actions to keys and thus this would lead to conflicts when accidently `j` key is mapped to an option. I would rather developer specify for their individual application what to do.

This change is in master and I will release soon new version.

**Comment by muellerj** [comment]
Sounds good to me. Can you leave a comment here, once the new version is released?

**Comment by piotrmurach** [comment]
Released `v0.8.0` and wrote some docs about [keyboard events](https://github.com/piotrmurach/tty-prompt#213-keyboard-events). Enjoy full Vim immersion!
